docs/api/instruments/hp/hp6641a.rst

```diff
@@ -0,0 +1,12 @@
+###########################################################
+System Power Supply (664xA, 665xA, 667xA, 668xA, and 669xA)
+###########################################################
+
+
+*Note:*
+	* Implemented HP6641A, HP6673A, HP6674A.
+
+
+.. autoclass:: pymeasure.instruments.hp.hp6641a.HP6641A
```

Comment:
Please add one of these class imports for each instrument you added.

This makes the above note superfluous.

Reply:
Added